numpy/lib/function_base.py

```diff
@@ -2681,14 +2681,14 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
     return c.squeeze()
 
 
-def _corrcoef_dispatcher(x, y=None, rowvar=None, bias=None, ddof=None, *,
+def _corrcoef_dispatcher(x=None, cov = None, y=None, rowvar=None, bias=None, ddof=None, *,
```

Comment:
The new parameter must be added at the end (or after the `*`, since order is irrelevant after that).  In principle we could add it last, but in practice we want users to write `cov=...` anyway.